api/src/main/java/org/openmrs/module/isanteplusreports/library/indicator/ReportIndicators.java

```diff
@@ -0,0 +1,15 @@
+package org.openmrs.module.isanteplusreports.library.indicator;
```

Comment:
@hmutaquiha please remove this whole class since it's meant to be used as an example